cmd/testdata/gotestsum-help-text

```diff
@@ -13,6 +13,7 @@ Flags:
       --jsonfile string                             write all TestEvents to file
       --jsonfile-timing-events string               write only the pass, skip, and fail TestEvents to the file
       --junitfile string                            write a JUnit XML file
+      --junitfile-always-include-output             include output even on successful tests
```

Comment:
Maybe `--junit-include-output=all` (with a default value of `failures`) or something like that?  "always" I think doesn't make a clear distinction between the two modes: "all output", and "only failure output".